accepted/color-4-new-spaces.md

```diff
@@ -631,20 +631,18 @@ Legacy colors that have [missing] components are
 
 ### Color Equality
 
-For determining *equality* between two colors:
+For determining *equality* between two colors `color1` and `color2`:
 
-* If both colors are [legacy colors](#legacy-color):
+* If both colors are [legacy colors](#legacy-color) in different spaces, return
+  `color.to-space(color1, rgb) == color.to-space(color2, rgb)`.
 
-  * Set each color to the result of [converting] the color into `rgb` space.
+* Otherwise, return whether the colors are in the same color space and their
+  channel and alpha values are fuzzy-equal, with [missing] channels treated as
+  0.
```

Comment:
Did this mean to say that missing channels are treated distinct than zero instead?